### PR TITLE
Updating the UMLS license informaton

### PR DESCRIPTION
**Pull request by bobsizemore**

### CODE REVIEW

**Review by dcwebermn — COMMENTED** [review]
Is the UMLS2022 update in scope for the 10/22 container edition release ?

**Comment by bobsizemore** [comment]
> Is the UMLS2022 update in scope for the 10/22 container edition release ?

I heard this was talked about during your platform meeting.  I do not know if it will make October, I would expect December is more reasonable.  We shipped in UMLS last November/December.  I assume we would not just merge the branch until it is needed.

**Review by dlangst — COMMENTED** [review]
Changes look good.  Merge in December-release timeframe.

File: src/pages/about/notices.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This program may reference National Library of Medicine (NLM).
 <q>The duplication, sale, or redistribution of NLM data must conform to fair use guidelines and copyright law.</q>
 
 SNOMED Clinical Terms®
-Version: SNOMEDCT_US_2021_03_01 March 1, 2021
+Version: SNOMEDCT_US_2022_03_01 March 1, 2022
 
 This Program includes SNOMED Clinical Terms® (SNOMED CT®) which is used by permission of the International Health Terminology Standards Development Organisation (IHTSDO). All rights reserved. SNOMED CT®, was originally created by The College of American Pathologists. <q>SNOMED</q> and <q>SNOMED CT</q> are registered trademarks of the IHTSDO.
```
Comment (dlangst) [inline src/pages/about/notices.md:28]: Do we need the two sentences at line 28 which are duplicates of the last two sentences at line 30?

File: src/pages/about/notices.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This program may reference National Library of Medicine (NLM).
...
```
Comment (dlangst) [inline src/pages/about/notices.md:29]: At line 30, it says "the vocabulary sources listed in the table below from the UMLS Metathesaurus, release 2021 AA.".  This should be 2022 AA?

File: src/pages/about/notices.md [hunk 3]
```diff
@@ -71,103 +73,104 @@ The program described in this document also includes the vocabulary sources list
...
 |TKMT2011|Traditional Korean Medical Terms, 2011 Korea, KOR|
-|USP_2021_02_15 (updated)|USP Compendial Nomenclature, February, 2021, United States Pharmacopeia, February, 2021|
+|USP_2022_02_15 (updated)|USP Compendial Nomenclature, February, 2022, United States Pharmacopeia, February, 2022|
 |USPMG_2020|USP Medicare Model Guidelines, Version 6, United States Pharmacopeia February 1, 2020|
 |UWDA173|Jose Mejino, M.D., University of Washington Digital Anatomist, (UWDA). Seattle (WA): University of Washington, Version 1.7.3, March, 2003.|
-|VANDF_2021_01_29 (updated)|Veterans Health Administration National Drug File, January 29, 2021 Washington, DC U.S. Department of Veterans Affairs|
+|VANDF_2022_01_31 (updated)|Veterans Health Administration National Drug File, January 31, 2022, Washington, DC U.S. Department of Veterans Affairs|
 
 Deidentified clinical records used in the development of this offering were provided by the i2b2 National Center for Biomedical Computing funded by U54LM008748 and were originally prepared for the Shared Tasks for Challenges in NLP for Clinical Data organized by Dr. Ozlem Uzuner, i2b2 and SUNY
```
Comment (dlangst) [inline src/pages/about/notices.md:176]: Can you double-check on a few of the mentioned dates starting on line 177...
1). "1995-2019" is mentioned in 2 diff places at line 177.  The current LOINC license "https://loinc.org/kb/license/" says 1995-2022, but these may be referring to different things.  At line 177, it splits it by "LOINC Table and LOINC codes" and by "LOINC Table, LOINC Table Core, LOINC Panels and Forms File...."
2). At the same website, under Notice of Third Party Content..." bullet E, says "1995-2020".  Should our line 179 in this document list the same years?
(The years listed on Line 181 and 183 match Bullets F and G, so those dates look correct).

**Review by dlangst — APPROVED** [review]
Updates look good to me.

File: src/pages/clouddocs/temporal_overview.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Temporal Overview (Preview)"
+title: "Temporal Overview"
```
Comment (dlangst) [inline src/pages/clouddocs/temporal_overview.md:2]: For the left navigation content column, you will also need to update (e.g. lines 162, 164) [https://github.com/merative/acd-containers/blob/master/src/data/nav-items.yaml](https://github.com/merative/acd-containers/blob/master/src/data/nav-items.yaml)

File: src/pages/clouddocs/release-notes.md [hunk 5]
```diff
@@ -22,6 +22,13 @@ subcollection: wh-acd
 
 The following sections document the new features and changes that were included for each release of the Annotator for Clinical Data service.
 
+## January 2023
+
+- Added support for UMLS 2022AA. UMLS 2019AA has been removed and 2020AA is now deprecated (will be removed in 2023 when we add support for UMLS 2023AA).
+- Added support to control whether the Hypothetical service uses internal triggers.
+- Added support for the ACD debug flag in the ACD Python SDK.
+- Updated the documentation for installing and upgrading the Container Edition.
```
Comment (cagrev) [inline src/pages/clouddocs/release-notes.md:30]: I don't think the last bullet is necessary. We are continually updating documentation.

**Review by cagrev — APPROVED** [review]
One comment. Otherwise, looks good to me. Thanks.

**Pull request merged**